packages/vue/src/components/FCheckboxField/FCheckboxField.spec.ts

```diff
@@ -13,11 +12,6 @@ function createWrapper({
         props: { value: "Default value", ...props },
         slots: { default: "Default label", ...slots },
         attachTo: createPlaceholderInDocument(),
-        global: {
-            provide: {
-                [getFieldsetLabelText]: () => undefined,
-            },
-        },
```

Comment:
This is removed because it is the default when injecting without a `FFieldset` present. No test-cases are affected by this.